src/lib/y2network/routing_table.rb

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+module Y2Network
+  class RoutingTable
+    attr_reader :routes
+
+    # MAIN_TABLE_ID = 254
+
+    def initialize(routes = [])
+      # @id = MAIN_TABLE_ID
+      @routes = routes
+    end
+
+    # @param route [Y2Network::Route] Route to add
```

Comment:
just curious why shortcut for this method and not others in enumerable for routes?

Reply:
For no particular reason. But yes, I guess we could convert this class into a `Enumerable`.

Reply:
then I think it will be only enhanced array. So the API stops to make sense for me. If we really want multiple routing tables, then I suggest to simple have routing table class that do mapping of ID and routes array and maybe some helper for default one, but keep just routes array and do not move it up to routing table itself.